backend/enhanced_eliza_agent_service.py

```
@@ -612,6 +631,21 @@ async def _build_single_utility(self, repo: Dict[str, Any]) -> UtilityResult:
             # Add necessary imports
             code = self._add_required_imports(code, repo_type)
 
+            # Check for Edge Function applicability
+            if self.edge_architect and self.edge_architect.analyze_task(repo_name):
+                try:
+                    edge_res = self.edge_architect.generate_function(
+                        repo_name.replace('_', '-'),
+                        f"Autonomous serverless capability for {repo_name}"
+                    )
+                    code += f"\n\n'''\n[EDGE ARCHITECT SUGGESTION]\n"
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)